third_party/terraform/tests/data_source_dns_managed_zone_test.go

```diff
@@ -24,6 +24,8 @@ func TestAccDataSourceDnsManagedZone_basic(t *testing.T) {
 					map[string]struct{}{
 						"dnssec_config.#":             {},
 						"private_visibility_config.#": {},
+						"peering_config.#":            {},
```

Comment:
If these are only ignored in GA we should wrap them in `unless ga` so that they are still tested in beta. That will also remind us to un-ignore them once they go GA.